utilities/utility.py

```diff
@@ -29,6 +40,21 @@ def validate_country_field(**kwargs):
         raise AttributeError("Not a valid country")
 
 
+def validate_name_field(**kwargs):
+    """
+    Function to validate location fields when
+    saving an object
+    :params kwargs
+    """
+    my_args = []
+    locations = ['Lagos', 'Kampala', 'Nairobi']
```

Comment:
@michael-basweti the logic here is not clear to me. What does the `my_args` variable contain?